docs/docs/components/reanimated_swipeable.md

```diff
@@ -81,27 +81,29 @@ an argument.
 
 ### `renderLeftActions`
 
-method that is expected to return an action panel that is going to be revealed from the left side when user swipes right.
-This map describes the values to use as inputRange for extra interpolation:
+accepts a function which receives the following arguments:
 
-SharedValue: [startValue, endValue]
+- `progress` value representing swiping progress relative to the width of the returned element.  
+  Equals `0` when `swipeable` is closed, `1` when left action is open.  
+  Linearly increases to `Infinity` as left action overshoots it's open position.
+- `translation` horizontal translation of the `swipeable` in pixels.
```

Comment:
```suggestion
- `translation` - a horizontal offset from the starting position.
```

Reply:
i would keep the `in pixels`, is that ok?

Reply:
Also, while `translation` implicitly suggests a global change, i think `offset` might sound like the change is relative to the action, so would maybe the following work better?
>- `translation` - a horizontal offset of the `swipeable` relative to it's closed position.

Reply:
Sounds good

Reply:
Is it in pixels? I think it will be in the scaled units (DIP on Android and Points on iOS) that take screen density into consideration

Reply:
I think we can use that if you want to keep the unit

Reply:
Alright, this is how it currently looks like, and i think it can remain that way, `offset` already implies `DIPs`:

>- `translation` - a horizontal offset of the `swipeable` relative to its closed position